### PR TITLE
OCD-3771: Add RWT collections page

### PR DESCRIPTION
**Pull request by andlar**

### CODE REVIEW

File: src/app/api/collections.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const useFetchRealWorldTestingCollection = ({
+  orderBy,
+  pageNumber,
+  pageSize,
+  sortDescending,
+  query,
+}) => {
+  const axios = useAxios();
+  return useQuery(['search/beta', orderBy, pageNumber, pageSize, sortDescending, query], async () => {
+    // const response = await axios.get(`/search/beta?rwtOptions=is_eligible,has_plans_url,has_results_url&${query}&pageNumber=${pageNumber}&pageSize=${pageSize}&orderBy=${orderBy}&sortDescending=${sortDescending}`); // option with no rwt filter
```
Comment (tmy1313) [inline src/app/api/collections.jsx]: Commented code.

**Review by tmy1313 — DISMISSED** [review]

**Review by tmy1313 — DISMISSED** [review]

File: e2e/pages/collections/real-world-testing/real-world-testing.readonly.spec.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      const searchTerm = 'Veracity';
+      page.searchForText(searchTerm);
+      expect(page.getTableCellText(page.results[0], columnIndex)).toContain(searchTerm);
+    });
+  });
+
+  xdescribe('when clicking on all RWT download button', () => {
+    it('should download a file', () => {
+      page.downloadAllRwtButton.click();
+      const apiFileName = 'APIDocData';
```
Comment (ashwinimore) [inline e2e/pages/collections/real-world-testing/real-world-testing.readonly.spec.js:93]: I understand that this test is marked to skip until the download is fully functional and the filename used here is just a placeholder until the filename is finalized in the implementation ticket, is that correct? if the idea of adding the test now is to enable the test by simply removing x, without making any other changes, then I think adding a filename that is relevant to RWT term is a good idea. I am fine with keeping this as-is and changing it in OCD-3804. I just had that thought when i saw APIDocData as sample text in RWT test

**Review by ashwinimore — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

File: e2e/pages/collections/real-world-testing/real-world-testing.readonly.spec.js (hunk 2 above)
Comment (andlar) [reply e2e/pages/collections/real-world-testing/real-world-testing.readonly.spec.js:93]: I don't think this button is going to exist at all once we get to that other ticket. I'd still like to leave it here as a placeholder for whatever we end up changing this to, but if that's a concern I can certainly remove it instead

**Review by athomas-git — DISMISSED** [review]

File: e2e/pages/collections/real-world-testing/real-world-testing.readonly.spec.js (hunk 2 above)
Comment (ashwinimore) [reply e2e/pages/collections/real-world-testing/real-world-testing.readonly.spec.js:93]: not a concern at all, leaving the test here until it's all finalized sounds good to me.

**Review by athomas-git — DISMISSED** [review]

**Review by ashwinimore — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by ashwinimore — DISMISSED** [review]

**Review by athomas-git — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by kekey1 — APPROVED** [review]

**Review by athomas-git — APPROVED** [review]

**Review by tmy1313 — APPROVED** [review]

**Review by ashwinimore — APPROVED** [review]

**Pull request merged**